### PR TITLE
Fix some overflowing test cases, issue 1120

### LINKED ISSUE #1120: Petaluma Font: Notes extend past the end of the measure.
Test Case: **Articulation - Accent/Tenuto › SVG + Petaluma**

When using the **Petaluma** font, the layout of quarter notes is incorrect, as it extends past the end of each measure.

Notes
* 1.2.93 does not support Petaluma.
* Broken in 3.0.9: `flow.html?ver=unpkg@3.0.9&module=Articulation&filter=accent`
* Broken in Master (2021-08-27): `flow.html?module=Articulation&filter=accent`

AccentTenuto

It is possible that the Petaluma glyphs take up too much horizontal space, and that we simply need to make the stave wider.
Or maybe we need to make the accent symbols take less horizontal space?

### PR DESCRIPTION
**Pull request by AaronDavidNewman**
resolves #1120 .  Fix a few cases where test cases were overflowing their container.  No rendering changes, just the test cases.  I'll be posting some diffs.

In most cases, I kept the original width if the hard-coded width fit, and only redid the ones that actually overflowed.  So the only visual diffs would be on tests that didn't fully appear, anyway.

For ChordChanges, I refactored to use factory and formatter, but there are no differences.

### CODE REVIEW

**Comment by AaronDavidNewman** [comment]
Articulation tests.
was:
image

is:
image

was:
image

is:
image

**Comment by AaronDavidNewman** [comment]
Accidental test cases.
was:
image

is:
image

**Comment by AaronDavidNewman** [comment]
Clef keys and Key Signature:
was:
image

is:
image

was;
image

is:
image

**Comment by AaronDavidNewman** [comment]
Formatting tests:
was:
image

is:
image

**Comment by AaronDavidNewman** [comment]
Ornament, only the jazz cases.  was;
image

is:
image

**Comment by AaronDavidNewman** [comment]
notehead:
was:
image
...
image

is:
image
...
image

**Comment by AaronDavidNewman** [comment]
last one.  stringnumber and StaveNote
was:
image

is:
image

was:
image
is:
image

File: tests/articulation_tests.ts [hunk 1]
```diff
@@ -274,7 +300,7 @@ function drawArticulations2(options: TestOptions, contextBuilder: ContextBuilder
   }
 
   // Helper function to justify and draw a 4/4 voice
-  Formatter.FormatAndDraw(ctx, staveBar4, notesBar4);
+  x += formatAndDrawToWidth(x, y, width, notesBar4, Barline.type.END, []);
```
Comment (rvilarl) [inline tests/articulation_tests.ts:303]: What is the aim of `x` it seems not to be used.

File: tests/stringnumber_tests.ts [hunk 2]
```diff
@@ -264,9 +267,11 @@ function multi(options: TestOptions): void {
...
 
 function drawAccidentals(options: TestOptions): void {
-  const f = VexFlowTests.makeFactory(options, 500);
-
-  const stave = f.Stave().setEndBarType(BarlineType.DOUBLE).addClef('treble');
+  const f = VexFlowTests.makeFactory(options, 750);
+  const glyphScale = 39; // default font scale
+  const musicStack = Tables.DEFAULT_FONT_STACK;
+  const clefWidth = Glyph.getWidth(musicStack, 'gClef', glyphScale); // widest clef
+  const score = f.EasyScore();
```
Comment (rvilarl) [inline tests/stringnumber_tests.ts:274]: It seems that `Score` is not used.

**Review by rvilarl — CHANGES_REQUESTED** [review]

File: tests/articulation_tests.ts (hunk 1 above)
Comment (AaronDavidNewman) [reply tests/articulation_tests.ts:303]: It is the first parameter to formatAndDrawToWidth in line 285

**Review by rvilarl — APPROVED** [review]

**Comment by 0xfe** [comment]
Thanks for cleaning these up, Aaron!

**Pull request merged**